src/main/java/core/basesyntax/service/impl/BalanceService.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;

```

Comment:
It (all OperationService implementations) should be subpackage strategy and you can name it impl because it is the implementation of your strategy